### DISCUSSION TITLE
Rivet 2 progress bar

### DISCUSSION THREAD

**Discussion by scottanthonymurray** [indiana-university/rivet-source#559] [Feature and enhancement requests]
Thread to discuss a progress bar component for Rivet 2.

**Comment by scottanthonymurray** [comment]
Some quick accessibility-related links:

- [MDN: Using the progressbar role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role)
- [W3 progressbar ARIA spec](https://www.w3.org/TR/wai-aria-1.1/#progressbar)
- [Web Accessibility for Developers: Progress Bars](https://pressbooks.library.ryerson.ca/wafd/chapter/progress-bars/)

**Comment by avitacco** [comment]
I have need of a progress bar in an application I'm writing. It takes files (of any given size) from a user and takes an action on them. The action is the same for all files, but can take between fractions of a second to upwards of an hour, depending on size. 

Having a progress bar that fits in with the rest of my rivet 2 site is highly desirable.
